spec/index.bs

```diff
@@ -4867,8 +4869,9 @@ dictionary GPUFragmentState: GPUProgrammableStage {
                 - |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/alpha}}
                     must be a [=valid GPUBlendComponent=].
             - |colorState|.{{GPUColorTargetState/writeMask}} must be &lt; 16.
-            - |descriptor|.{{GPUProgrammableStage/module}} must contain an output variable that:
-                - is [=statically used=] by |descriptor|.{{GPUProgrammableStage/entryPoint}}, and
+            - |descriptor|.{{GPUProgrammableStage/entryPoint}} must have a [=pipeline output=] value:
```

Comment:
The reverse should also be checked. Each pipeline output must have location that matches a target colorState.

Reply:
That seems to match what @austinEng found in #1900 although we can probably fix this in a follow-up PR.

Reply:
Yeah, let's focus on the vertex data in this PR